inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -683,9 +683,6 @@
                     "id": {
                         "type": "string"
                     },
-                    "set": {
```

Comment:
I didn't know we could have multiple types! Yeah, this will solve the problem and it will leave `set` there.